swift/build_ffi.sh

```diff
@@ -85,7 +90,12 @@ if [[ -n "${DEVELOPER_SDK_DIR:-}" ]]; then
   export LIBRARY_PATH="${DEVELOPER_SDK_DIR}/MacOSX.sdk/usr/lib:${LIBRARY_PATH:-}"
 fi
 
-echo_then_run cargo build -p libsignal-ffi ${RELEASE_BUILD:+--release} ${VERBOSE:+--verbose}
+BUILD_CMD=cargo
+if [[ -n "${USE_XARGO:-}" ]]; then
+	BUILD_CMD=xargo
```

Comment:
I originally tried to use the `-Zbuild-std` approach, but it resulted in a lot of problems with some of our dependencies. Using `xargo` works around the issues.